contracts/libs/TokenRouter.sol

```diff
@@ -61,6 +62,14 @@ abstract contract TokenRouter is GasRouter {
         emit SentTransferRemote(_destination, _recipient, _amountOrId);
     }
 
+    function transferRemote(
```

Comment:
Nice sneaky change

Reply:
It is nice but it may cause confusion that some Hyp contracts take bytes32 and others take address

Reply:
reverting